wled00/json.cpp

```diff
@@ -487,9 +487,7 @@ bool deserializeState(JsonObject root, byte callMode, byte presetId)
     }
   }
 
-  if (root.containsKey(F("np"))) {
-    doAdvancePlaylist = root[F("np")].as<bool>(); //advances to next preset in playlist when true
-  }
+  doAdvancePlaylist = root["np"].as<bool>() || doAdvancePlaylist; //advances to next preset in playlist when true
```

Comment:
It should be single `|` not double `||`. Single `|` operator is overloaded. You also have to omit `.as<bool>()`.

Reply:
ok, updated